spirv_cross/src/spirv.rs

```diff
@@ -363,6 +363,7 @@ where
         &mut self,
         options: &<Self as Compile<TTarget>>::CompilerOptions,
     ) -> Result<(), ErrorCode> {
+        self.compiler.has_been_compiled = false;
```

Comment:
yes, this line still makes sense though

Reply:
Why do we need it here? `has_been_compiled` is set to `false` during `parse`, and we can't safely reuse compilers. We also can't depend on this flag for `get_cleansed_entry_point_name` if we changed it here, because `set_compiler_options` could reset it at any point

Reply:
I think the fact that we can't compile multiple times after parse is not by design, it's just lacking a proper cleanup there. Anyhow, this line serves no goal at this moment, so I'll remove it.